cime_config/cesm/machines/config_machines.xml

```diff
@@ -868,7 +868,10 @@
     <DESC>"NCAR CGD Linux Cluster 48 pes/node, batch system is PBS"</DESC>
     <NODENAME_REGEX>hobart</NODENAME_REGEX>
     <COMPILERS>intel,pgi,nag,gnu</COMPILERS>
-    <MPILIBS>mvapich2,openmpi,mpi-serial</MPILIBS>
```

Comment:
Is this change backwards compatible? Meaning we don't have to change mpilibs for all machines?